tests/test_bot.py

```diff
@@ -1273,37 +1282,35 @@ def test_send_game_default_allow_sending_without_reply(self, default_bot, chat_i
                     chat_id, game_short_name, reply_to_message_id=reply_to_message.message_id
                 )
 
-    @flaky(3, 1)
-    def test_set_game_score_1(self, bot, chat_id):
+    @xfail
+    def test_set_game_score_1(self, game_bot, chat_id):
         # NOTE: numbering of methods assures proper order between test_set_game_scoreX methods
+        # First, test setting a score.
+        game_short_name = 'test_game'
+        game = game_bot.send_game(chat_id, game_short_name)
 
-        def func():
-            game_short_name = 'test_game'
-            game = bot.send_game(chat_id, game_short_name)
-
-            message = bot.set_game_score(
-                user_id=chat_id,
-                score=int(BASE_TIME) - HIGHSCORE_DELTA,
-                chat_id=game.chat_id,
-                message_id=game.message_id,
-            )
-
-            assert message.game.description == game.game.description
-            assert message.game.animation.file_id == game.game.animation.file_id
-            assert message.game.photo[0].file_size == game.game.photo[0].file_size
-            assert message.game.text != game.game.text
+        message = game_bot.set_game_score(
+            user_id=chat_id,
+            score=60,
+            chat_id=game.chat_id,
+            message_id=game.message_id,
+        )
 
-        expect_bad_request(func, 'Bot_score_not_modified', 'This test is a diva for some reason.')
+        # We don't test for animation file id, since it actually changes when the score is updated
```

Comment:
can't we just test with `file_unique_id`?

Reply:
yea sure, will add